src/utils.ts

```diff
@@ -234,6 +245,19 @@ export namespace Utils {
           "The transaction has operations that are unrecognized",
         );
       }
+      if (op.value === undefined) {
+        throw new InvalidSep10ChallengeError(
+          "The transaction's operation values should not be null",
+        );
+      }
+      if (
+        op.name === "web_auth_domain" &&
+        op.value.compare(Buffer.from(webAuthDomain))
```

Comment:
Love it. 🚀 

Initially I wasn't sure if this would work because `compare` returns a `-1`, `0`, or `1` depending how the two buffers sort, but TIL it does work because:
```javascript
> !!-1
true
> !!1
true
> !!0
false
```